packages/core/admin/ee/server/src/services/audit-logs.ts

```diff
@@ -62,7 +62,9 @@ const getEventMap = (defaultEvents: any) => {
 };
 
 const getRetentionDays = (strapi: Core.Strapi) => {
-  const licenseRetentionDays = strapi.ee.features.get('audit-logs')?.options.retentionDays;
+  const featureConfig = strapi.ee.features.get('audit-logs');
+  const licenseRetentionDays =
+    typeof featureConfig === 'object' && featureConfig?.options.retentionDays;
```

Comment:
Actually there is a type error here on v5/main since options can't exist on a potential string. The type comes from [here](https://github.com/strapi/strapi/blob/9d4475b11a4c817c5c045566941ae80d17cbf560/packages/core/types/src/core/strapi.ts#L41-L49). Not crazy about this so if you have another idea lmk. Also do you have any idea what features would return a string? I thought they all returned `{name: string options: any}`


![Screenshot 2024-05-03 at 09 40 16](https://github.com/strapi/strapi/assets/26598053/a4b4a86c-cc93-42fd-ae8a-3605f4226c6d)

Reply:
Don't know about any features that use this, but let's keep your syntax then 👍